styles.css

```diff
@@ -547,3 +549,182 @@ a {
   margin-left: 130px;
   margin-top: -50px;
 }
+
+/* Contact Form section */
+
+.form_section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 120px;
+  padding-left: 24px;
+  padding-right: 24px;
+  gap: 58px;
+  background: #fff;
+  margin-top: 1650px;
+}
+
```

Comment:
- [ ] [OPTIONAL] Please fix the horizontal scrolling as horizontal scrolling involves more effort since users typically have to move their mouse pointer to a specifically designated area on the screen and drag (while holding the mouse button) or click. In addition to this, some horizontal scrolling implementations make the design judder and hence impair user experience. 
 
 It can be caused by different factors. It could occur because of unexpectedly wide content or a fixed-width element (px) that is wider than the viewport. In order to fix this I would suggest you to set the overflow-x value to hidden, and use responsive value like %, em, rem instead of fixed values like px.



https://user-images.githubusercontent.com/44798044/228441886-819a5804-25ac-4777-8ce5-2dc6b8214fab.mov